app/controllers/random_thoughts_controller.rb

```diff
@@ -16,9 +18,18 @@ def create
     render 'show', status: :created
   end
 
+  def update
+    @random_thought.update(random_thought_params)
```

Comment:
This should be `@random_thought.update!(random_thought_params)` so that it throws an exception if there is an issue updating the record instead of `update(...` which fails silently.  Since there are currently no validations, this is a system faulting (e.g. database fault) edge case, but still a possibility.